patroni/ha.py

```diff
@@ -1083,7 +1089,7 @@ def restart(self, restart_data, run_async=False):
                 return (False, 'restart failed')
 
     def _do_reinitialize(self, cluster):
-        self.state_handler.stop('immediate')
+        self.state_handler.stop('immediate', stop_timeout=self.master_stop_timeout())
```

Comment:
This code-path is never executed when postgres is running as master, but IMO it still makes sense to pass non-zero value here. It would theoretically help with the case when postgres doesn't respond to immediate shutdown.

Reply:
Agree, Do you have any number in mind? I am leaning towards 1 minute timeout provided that the call is intended for reinitialize. Thanks.